### ISSUE TITLE
Add documentation for connecting to a replica set

### ISSUE THREAD

**Issue by paton** [andzdroid/mongo-express#106]
#105

**Comment by wulfsolter** [comment]
Bump... @paton

**Comment by jamrizzi** [comment]
Yes, can someone please document this. I can't figure out how to connect mongo-express to a replica set.

**Comment by dozoisch** [comment]
@jamrizzi https://github.com/mongo-express/mongo-express/pull/105#issuecomment-144875600

**Comment by heartnetkung** [comment]
Is it sufficient to document this only in config.default.js as a comment? 

I think the replica set support is still a hack since it's not supported in environment variable and all. We could document it in readme.md once someone fully implement the replica support.

**Comment by jamrizzi** [comment]
Could you also add support for this in the docker container?  Maybe have it pull from a comma delimited list of the replica sets in an environment variable.

**Comment by heartnetkung** [comment]
@jamrizzi i think I can do this tomorrow.

**Comment by jamrizzi** [comment]
Actually, that shouldn't be that hard to do.  If it makes things easier, I'll do a pull request.

**Comment by jamrizzi** [comment]
The pull request is at https://github.com/mongo-express/mongo-express/pull/284.

**Comment by shakaran** [comment]
@paton @heartnetkung @wulfsolter @clayrisser 

This issue is old, but as far I undestand it is finished. Could be the issue closed or it needs more work to do? Thanks
